app.rb

```diff
@@ -1,9 +1,37 @@
 require 'sinatra/base'
+require_relative './lib/message_handler'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/message_page' do 
```

Comment:
Typically, I would expect the messages to be shown on the index page (the `/` route) of the app as this is where the user lands by default. If you want to keep the `/message_page` route, you should have a redirect that will take the user from`/` to `/message_page`.